packages/doctype-tile-handler/src/__tests__/three-id.test.ts

```diff
@@ -0,0 +1,314 @@
+import CID from 'cids'
```

Comment:
same as before, I'm not reviewing this file because I assume it's just code motion with no changes. If anything meaningful changed please LMK